Tensile/KernelWriterAssembly.py

```diff
@@ -1127,24 +1128,12 @@ def initKernel(self, kernel, tPA, tPB ):
     # special cases : hgemm  +  HPA ([H/H/H] compute = f16, but internal = f32)
     self.bpeCinternal = int(self.bpr * kernel["ProblemType"]["ComputeDataType"].numRegisters())
 
-    # Dexternal = the "current" kernel output type,
-    # - default: the "current" kernel is a non-GSU-kernel,
-    #     Dexternal = Cexternal (= DestDataType) and is the final gemm result
-    #
-    # - For GSU: the "current" kernel is a GSU-kernel,
-    #     this kernel returns a temp buffer with same type as Cinternal.
-    #     Later, another kernel will accumulate this buffer
-    #     and convert the final result to Cexternal (= DestDataType) as the gemm result
-    self.bpeDexternal = int(self.bpr * kernel["ProblemType"]["DestDataType"].numRegisters())
-
-    # TODO- Currently, Cexternal, Cinteral, Dexternal are kind of confusing, need to clean up
-
```

Comment:
Dexternal is removed, but could we still put the original comments here to account for "Cexternal" ?
```
    # Cexternal = the "current" kernel output type,
    # - default: the "current" kernel is a non-GSU-kernel,
    #     Cexternal (= DestDataType) and is the final gemm result
    #
    # - For GSU: the "current" kernel is a GSU-kernel,
    #     this kernel returns a temp buffer with same type as Cinternal.
    #     Later, another kernel will accumulate this buffer
    #     and convert the final result to DestDataType as the gemm result
```

Reply:
done